common/app/contentapi/ContentApiClient.scala

```diff
@@ -98,10 +98,9 @@ trait ApiQueryDefaults extends Logging {
 trait MonitoredContentApiClientLogic extends ContentApiClientLogic with ApiQueryDefaults with Logging {
 
   val httpClient: HttpClient
-  val _httpClient = httpClient //TODO: to delete once ContentApiClient fully uses DI
```

Comment:
Is there a particular reason for the `httpClient` to require stability (i.e. being a `val`, not a `def`)?